mesonbuild/mintro.py

```diff
@@ -453,13 +455,12 @@ def run(options: argparse.Namespace) -> int:
 
     if 'meson.build' in [os.path.basename(options.builddir), options.builddir]:
         # Make sure that log entries in other parts of meson don't interfere with the JSON output
-        mlog.disable()
-        backend = backends.get_backend_from_name(options.backend)
-        assert backend is not None
-        intr = IntrospectionInterpreter(sourcedir, '', backend.name, visitors = [AstIDGenerator(), AstIndentationGenerator(), AstConditionLevel()])
-        intr.analyze()
-        # Re-enable logging just in case
-        mlog.enable()
+        with redirect_stdout(sys.stderr):
```

Comment:
I had no idea this even existed.

Reply:
Python comes with batteries included. You can even import antigravity (but you cannot import braces from future).

Reply:
I did not know this trick either, very nice!